python/cudf/cudf/io/csv.py

```diff
@@ -61,9 +61,17 @@ def read_csv(
         path_or_data=filepath_or_buffer,
         compression=compression,
         iotypes=(BytesIO, StringIO),
+        byte_ranges=[byte_range] if byte_range else None,
+        clip_dummy_buffer=True if byte_range else False,
         **kwargs,
     )
 
+    # Adjust byte_range for clipped dummy buffers
```

Comment:
```suggestion
    # Adjust byte_range for clipped placeholder buffers
```

or any of the alternatives suggested in https://developers.google.com/style/word-list#dummy-variable

Reply:
If possible (i.e., if we're not breaking Pandas API compat), we should replace all occurrences.

Reply:
Thanks for the quick review here @shwina !! I really appreciate it. I'll change dummy to placeholder throughout

Reply:
Okay - Ended up going with "dummy" -> "local"